### PR TITLE
how-to/upgrade: Fix upgrade of v3.0.0 some errors attention.

### PR DESCRIPTION
**Pull request by superlzs0476**
### What is changed, added or deleted? 
docs-cn/v3.0/how-to/upgrade/from-previous-version.md

### Which version does your change affect? 
v3.0.0

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by superlzs0476** [comment]
@liubo0127  @lilin90  PTAL

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 1]
```diff
@@ -138,13 +138,17 @@ $ ansible-playbook local_prepare.yml
...
-> 为优化 TiDB 集群组件的运维管理，TiDB 3.0 版本对 `systemd` 模式下的 `PD service` 名称进行了调整。与之前版本相比，滚动升级 TiDB 3.0 版本集群组件的操作略有不同，注意升级前后 `process_supervision` 参数配置须保持一致。
+> 为优化 TiDB 集群组件的运维管理，TiDB 3.0 版本对 `systemd` 模式下的 `PD service` 名称进行了调整。如当前版本**小于** TiDB 3.0 版本，滚动升级到 TiDB 3.0 版本集群组件的操作略有不同，注意升级前后 `process_supervision` 参数配置须保持一致。
+
 
 如果 `process_supervision` 变量使用默认的 `systemd` 参数，则通过 `excessive_rolling_update.yml` 滚动升级 TiDB 集群。
 
 ```
 $ ansible-playbook excessive_rolling_update.yml
 ```
+>  **注意：**
```
Comment (liubo0127) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion

>  **注意：**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 2]
```diff
@@ -138,14 +138,19 @@ $ ansible-playbook local_prepare.yml
...
+> 为优化 TiDB 集群组件的运维管理，TiDB 3.0 版本对 `systemd` 模式下的 `PD service` 名称进行了调整。如当前版本**小于** TiDB 3.0 版本，滚动升级到 TiDB 3.0 版本集群组件的操作略有不同，注意升级前后 `process_supervision` 参数配置须保持一致。
+
 
 如果 `process_supervision` 变量使用默认的 `systemd` 参数，则通过 `excessive_rolling_update.yml` 滚动升级 TiDB 集群。
 
 ```
 $ ansible-playbook excessive_rolling_update.yml
 ```
 
+>  **注意：**
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
> **注意：**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 3]
```diff
@@ -138,14 +138,19 @@ $ ansible-playbook local_prepare.yml
...
+> 
+> 如当前版本**大于或等于** TiDB 3.0 版本在滚动升级及日常滚动重启 TiDB 集群仍然使用 `rolling_update.yml` 操作。
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
> 如当前版本**大于或等于** TiDB 3.0 版本，则滚动升级及日常滚动重启 TiDB 集群仍然使用 `rolling_update.yml` 操作。
```

**Comment by yikeke** [comment]
Please also update the `v3.0` changes to the `dev` folder. @superlzs0476

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 4]
```diff
@@ -138,14 +138,19 @@ $ ansible-playbook local_prepare.yml
 
 > **注意：**
 >
-> 为优化 TiDB 集群组件的运维管理，TiDB 3.0 版本对 `systemd` 模式下的 `PD service` 名称进行了调整。与之前版本相比，滚动升级 TiDB 3.0 版本集群组件的操作略有不同，注意升级前后 `process_supervision` 参数配置须保持一致。
+> 为优化 TiDB 集群组件的运维管理，TiDB 3.0 版本对 `systemd` 模式下的 `PD service` 名称进行了调整。如当前版本**小于** TiDB 3.0 版本，滚动升级到 TiDB 3.0 版本集群组件的操作略有不同，注意升级前后 `process_supervision` 参数配置须保持一致。
+
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 5]
```diff
@@ -138,14 +138,19 @@ $ ansible-playbook local_prepare.yml
...
 
 如果 `process_supervision` 变量使用默认的 `systemd` 参数，则通过 `excessive_rolling_update.yml` 滚动升级 TiDB 集群。
 
 ```
 $ ansible-playbook excessive_rolling_update.yml
 ```
 
+> **注意：**
+>
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
>
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB 3.0 升级操作指南
 category: how-to
+aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/']
```
Comment (yikeke) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
```
We only need `aliases` in `v3.0`. Remember to remove this line when you copy the doc from v3.0.

**Comment by yikeke** [comment]
PTAL @liubo0127

**Comment by liubo0127** [comment]
LGTM

**Pull request merged**